packages/sanes-chrome-extension/src/extension/background/model/signingServer/index.server.unit.spec.ts

```diff
@@ -1,4 +1,5 @@
-import { JsonRpcSigningServer, SignAndPostAuthorization, TransactionEncoder } from '@iov/multichain';
+import { TransactionEncoder } from '@iov/encoding';
+import { JsonRpcSigningServer, SignAndPostAuthorization } from '@iov/multichain';
```

Comment:
You said that
> @iov/multichain is not needed anymore by the web apps

Reply:
This is code of the browser extension. In the extension, @iov/multichain is still used